plugin/environment.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var ErrNotFound = errors.New("Item not found")
```

Comment:
@ali-farooq0, thoughts?

Reply:
@jfrerich We went with a generic `item not found` error message for the `plugin` package, as opposed to having multiple entity specific error messages `manifest not found`, `bundle not found`, `plugin not found`. Most of the time you'll know which entity you're attempting to retrieve so it won't be ambiguous when you get this message.

This is also part of streamlining apis that ought to return an error when an entity is not found. If need be we can introduce entity specific errors, since this is all internal to our server, it'll be easier to change in the future.

Reply:
Thanks for the clarification, @ali-farooq0.  It was just a 0/5 nit :)